daily_milestone.go

```diff
@@ -158,34 +177,42 @@ func getMilestones(baseURL string, token string, projectID string) ([]simpleMile
 func createMilestoneData(advance int, timeInterval string) []simpleMilestone {
 	today := time.Now().Local()
 	list := []simpleMilestone{}
-	for i := 0; i < advance; i++ {
-		if timeInterval == "daily" {
+
+	switch {
+	case timeInterval == "daily":
+		for i := 0; i < advance; i++ {
 			date := today.AddDate(0, 0, i).Format("2006-01-02")
 			milestone := simpleMilestone{}
 			milestone.Title = date
 			milestone.DueDate = date
 			list = append(list, milestone)
-		} else if timeInterval == "weekly" {
-			today = today.AddDate(0, 0, 7)
```

Comment:
I might be missing it, but in your current implementation you are not iterating through the next weeks, but only ever get one week back right?

Reply:
I am taking it back to the current week's monday and then iterating it through the next weeks